Tella/Components/TrailingButtonToolbar.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  Copyright © 2023 HORIZONTAL. All rights reserved.
+//
+
+import SwiftUI
+
+struct TrailingButtonToolbar: ToolbarContent {
+    
+    var title : String = ""
+    var completion : (() -> ())?
+    
+    var body: some ToolbarContent {
+       
+        ToolbarItem(placement: .navigationBarTrailing) {
+           
+            Button {
+                completion?()
+            } label: {
+                Text(title)
+                .font(.custom(Styles.Fonts.regularFontName, size: 14))
+                .foregroundColor(Color.white)
+                .frame(width: 260,height:25,alignment:.trailing)
```

Comment:
Are these values going to be always the same? I'm wondering if it would make sense to pass these values by params to make it more flexible in case we need it in the future.

Reply:
In this case I'll remove the width parameter and keep the height